app/products/calls/screens/call_screen/index.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import withObservables from '@nozbe/with-observables';
+import {combineLatest, of as of$} from 'rxjs';
+import {switchMap} from 'rxjs/operators';
+
+import CallScreen from '@calls/screens/call_screen/call_screen';
+import {observeCurrentCall} from '@calls/state';
+import {CallParticipant} from '@calls/types/calls';
+import DatabaseManager from '@database/manager';
+import {observeTeammateNameDisplay, observeUsersById} from '@queries/servers/user';
+
+const enhanced = withObservables([], () => {
+    const currentCall = observeCurrentCall();
+    const database = currentCall.pipe(
+        switchMap((call) => of$(call ? call.serverUrl : '')),
+        switchMap((url) => of$(DatabaseManager.serverDatabases[url]?.database)),
```

Comment:
This is an edge case.  

It's safer to use `getServerDatabaseAndOperator` from the DatabaseManager to retrieve the database or operator.  But that way also works. 

What happens if database is undefined ( for any reason ) ? We have multi-server support and in the event the user logged onto ServerA on the mobile app only and not ServerB, can he still receive a call from ServerB on the app?

Reply:
If they're not logged into server B, then they can't receive a call from server B.  As for what if the database is undefined? Good question -- below we're using ternaries to guard against it.

Reply:
I rather keep it like this cause the function @avinashlng1080 is proposing throws and this way it doesn't